tests/timetables/test_trigger_timetable.py

```diff
@@ -543,6 +543,39 @@ def test_multi_run_next(last, expected):
     assert next_info == DagRunInfo.exact(expected)
 
 
+@pytest.mark.parametrize(
+    "last, current_time, expected",
+    [
+        pytest.param(
+            pendulum.datetime(2025, 2, 1),
+            pendulum.datetime(2025, 2, 1, minute=30),
+            pendulum.datetime(2025, 2, 1, minute=30),
+            id="exact",
+        ),
+        pytest.param(
+            pendulum.datetime(2025, 2, 1),
+            pendulum.datetime(2025, 2, 1, minute=59),
+            pendulum.datetime(2025, 2, 1, minute=30),
```

Comment:
```suggestion
            pendulum.datetime(2025, 2, 1, hour=1, minute=59),
            pendulum.datetime(2025, 2, 1, hour=1, minute=30),
```
I think a better test would be that there have actually been several missed runs since last